public/components/datasources/components/__tests__/testing_constants.ts

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Not sure how this got pulled in, should be on main already, can you double check/merge